### PR TITLE
Remove conteúdo em inglês do README para melhorar a organização

### PR DESCRIPTION
**Pull request by rnihesh**
English- I've updated to readme, taken out the English version.
Portuguese- Atualizei para o leia-me e removi a versão em inglês.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,40 @@
 
-
 # O QUE É a comuREDE?
 
-UM NEGÓCIO DE IMPACTO SOCIAL OPEN SOURCE
+UM NEGÓCIO DE IMPACTO SOCIAL OPEN SOURCE
```
Comment (lhsazevedo) [inline README.md:4]: This trailing white space is not needed, revert it please

File: README.md [hunk 2]
```diff
@@ -1,50 +1,40 @@
...
-[WATCH HERE](https://youtu.be/DjV8mJWbhzo)
-
-
 # COMO FUNCIONA?
-   
+
 ### core
 
-Versão 0.18 e 0.20 - back / frontend
+Versão 0.18 e 0.20 - back / frontend
```
Comment (lhsazevedo) [inline README.md:13]: Yeah, these trailing spaces makes sense but I find them hard to maintain (it doesn't show up for some people when editing).

What about adding a blank line? Like this:

```md
Versão 0.18 e 0.20 - back / frontend

PHP 5 / HTML, CSS e JAVASCRIPT
```

File: README.md [hunk 3]
```diff
@@ -1,50 +1,40 @@
...
 [Design](https://www.dropbox.com/scl/fi/l7tacgtlutw8a9ec8qzkr/Estiliza-o_comuREDE-por-Pedro-Rangel_LABIC-2017.pdf?rlkey=a7xptpwqqin727s11s9c2piw5&st=70aupz03&dl=0) por Pedro Rangel
 
 [Topologia (em produção)](https://www.dropbox.com/scl/fi/kr0lbizr523plri21ft8h/Projeto-L-gico-comuREDE_030417.pdf?rlkey=ftf5a14h37tvash42hscbafu6&st=uqpw0wcd&dl=0) por Filipe Rimes    
-   
+
 ## hardware / software
 
-### servidor
-Raspberry W Zero (para testes-dev ou [VM](https://sourceforge.net/projects/osboxes/files/v/vb/48-R-pb/2019-04-11/R20190411VB-32bit.7z/download) "Username: pi Password: osboxes.org") / Rasp 3 (em produção) / Raspbian / Apache / MQTT / MariaDB 
+### servidor
```
Comment (lhsazevedo) [inline README.md:25]: This trailing white space is not needed, revert it please

File: README.md [hunk 4]
```diff
@@ -1,50 +1,40 @@
...
 
 [Topologia (em produção)](https://www.dropbox.com/scl/fi/kr0lbizr523plri21ft8h/Projeto-L-gico-comuREDE_030417.pdf?rlkey=ftf5a14h37tvash42hscbafu6&st=uqpw0wcd&dl=0) por Filipe Rimes    
-   
+
 ## hardware / software
 
-### servidor
-Raspberry W Zero (para testes-dev ou [VM](https://sourceforge.net/projects/osboxes/files/v/vb/48-R-pb/2019-04-11/R20190411VB-32bit.7z/download) "Username: pi Password: osboxes.org") / Rasp 3 (em produção) / Raspbian / Apache / MQTT / MariaDB 
+### servidor  
+Raspberry W Zero (para testes-dev ou [VM](https://sourceforge.net/projects/osboxes/files/v/vb/48-R-pb/2019-04-11/R20190411VB-32bit.7z/download) "Username: pi Password: osboxes.org") / Rasp 3 (em produção) / Raspbian / Apache / MQTT / MariaDB
```
Comment (lhsazevedo) [inline README.md:26]: This trailing white space is not needed, revert it please

File: README.md [hunk 5]
```diff
@@ -1,50 +1,40 @@
...
+
 ## hardware / software
 
-### servidor
-Raspberry W Zero (para testes-dev ou [VM](https://sourceforge.net/projects/osboxes/files/v/vb/48-R-pb/2019-04-11/R20190411VB-32bit.7z/download) "Username: pi Password: osboxes.org") / Rasp 3 (em produção) / Raspbian / Apache / MQTT / MariaDB 
+### servidor  
+Raspberry W Zero (para testes-dev ou [VM](https://sourceforge.net/projects/osboxes/files/v/vb/48-R-pb/2019-04-11/R20190411VB-32bit.7z/download) "Username: pi Password: osboxes.org") / Rasp 3 (em produção) / Raspbian / Apache / MQTT / MariaDB  
 
-### sensores autônomos
+### sensores autônomos
```
Comment (lhsazevedo) [inline README.md:28]: This trailing white space is not needed, revert it please

File: README.md [hunk 6]
```diff
@@ -1,50 +1,40 @@
...
 
 [Apresentação Sensor](https://www.dropbox.com/scl/fi/vea61m6vk0lcxz9zgk2yk/Apresenta-o_Sensor-Aut-nomo-Fluxo-de-gua_comuREDE_AGO18_LQ.pdf?rlkey=ga2att2d75l0blcjq36lofrg5&st=11yt847d&dl=0) (AGO18)
-              
+
 [Sketch Sensor Fluxo v0.20](https://www.dropbox.com/s/a0mwp6wxfzbavyx/Sensor_Auton_v02_AGUAf_LIME-CLOUDDEV_Node_ID24130-89_190120-B.zip?dl=0) (SET19)
 
 [Sketch Sensor v0.18](https://olha.ai/bYlY1) (AGO18)
 
-### banco de dados
+### banco de dados
```
Comment (lhsazevedo) [inline README.md:37]: This trailing white space is not needed, revert it please

File: README.md [hunk 7]
```diff
@@ -53,21 +43,21 @@ MySQL
 
 [Modelo Entidade Relacional v0.10](https://olha.ai/2fUSM) (SET18)
 
-### ciência de dados
+### ciência de dados
```
Comment (lhsazevedo) [inline README.md:46]: This trailing white space is not needed, please revert it

File: README.md [hunk 8]
```diff
@@ -53,21 +43,21 @@ MySQL
...
+> [Red Bull Amaphiko](https://www.facebook.com/RedBullAmaphiko/videos/751140301987210/?v=751140301987210)  
 
-> [Olhar Digital](https://olhardigital.com.br/video/residencia-hacker-impulsiona-projetos-de-impacto-social-conheca-as-ideias/89008)
+> [Olhar Digital](https://olhardigital.com.br/video/residencia-hacker-impulsiona-projetos-de-impacto-social-conheca-as-ideias/89008)  
 
-> [Canal Futura](http://www.futuraplay.org/video/comurede-e-drone-para-prevencao-de-dengue/501921/)
+> [Canal Futura](http://www.futuraplay.org/video/comurede-e-drone-para-prevencao-de-dengue/501921/)  
 
-> [Instituto Federal do Rio de Janeiro](https://portal.ifrj.edu.br/laboratorio-informatica-ruas)
+> [Instituto Federal do Rio de Janeiro](https://portal.ifrj.edu.br/laboratorio-informatica-ruas)
```
Comment (lhsazevedo) [inline README.md:63]: These trailing white space is not needed, please revert it

**Review by lhsazevedo — CHANGES_REQUESTED** [review]
Nice! Thank you for fixing the white space as well :)

Some of them though aren't needed, I've marked them with comments.
Could fix them so we can proceed with the merge?

**Comment by rnihesh** [comment]
Yeah I'll make it

**Comment by rnihesh** [comment]
I made the changes

**Comment by lhsazevedo** [comment]
Awesome, thanks! :rocket:

**Pull request merged**